packages/components/psammead-brand/src/index.jsx

```diff
@@ -100,18 +108,20 @@ const StyledBrand = ({
   <Fragment>
```

Comment:
Also, this doesnt work when the svg is a link weirdly :P May need to move the wrapper up a bit